helm_deploy/apply-for-legal-aid/templates/deployment_web.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "apply-for-legal-aid.fullname" . }}
+  labels:
+    app: {{ template "apply-for-legal-aid.name" . }}
+    chart: {{ template "apply-for-legal-aid.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "apply-for-legal-aid.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "apply-for-legal-aid.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: clamav
+          image: "926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-apply-for-legal-aid/clamav:1.0.0"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: clamav
+              containerPort: 3310
+              protocol: TCP
+          resources:
+{{ toYaml .Values.clamav.resources | indent 12 }}
+        - name: metrics
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: IfNotPresent
+          command: ['sh', '-c', "bundle exec prometheus_exporter"]
```

Comment:
Just out of interest - is there a reason why this `command` can use a string with spaces, where other's seem to have to split the command words. Would this work:
```
command: ['sh', '-c', "bundle", "exec", "prometheus_exporter"]
```
And more interestingly - would this work:
```
command: ['sh -c bundle exec prometheus_exporter']
```
Because if that works lets stop messing around with arrays and just pass the commands in as a single string.

Reply:
Weird but neither `['sh', '-c', 'bundle', 'exec', 'prometheus_exporter']` or `['sh -c bundle exec prometheus_exporter']` works. It needs to be `['sh', '-c', 'bundle exec prometheus_exporter']`

Reply:
Weird in deed. Thank you for trying.